addons/binding/org.openhab.binding.homematic/README.md

```diff
@@ -140,20 +148,20 @@ If you really like to manually configure a thing:
 ```
 Bridge homematic:bridge:ccu [ gatewayAddress="..." ]
 {
-  Thing HM-LC-Dim1T-Pl-2    JEQ0999999
+  Thing HM-LC-Dim1T-Pl-2 MyDevice [address="JEQ0999999"]
```

Comment:
You should mention the new address parameter somewhere above (not merely in the examples).